pkg/network/tracer/tracer_classification_test.go

```diff
@@ -1725,11 +1729,20 @@ func testEdgeCasesProtocolClassification(t *testing.T, tr *Tracer, clientHost, t
 				c, err := defaultDialer.DialContext(timedContext, "tcp", ctx.targetAddress)
 				require.NoError(t, err)
 				defer c.Close()
-				c.Write([]byte("GET /200/foobar HTTP/1.1\n"))
-				io.Copy(io.Discard, c)
-				// http2 prefix.
-				c.Write([]byte("PRI * HTTP/2.0\r\n\r\nSM\r\n\r\n"))
-				io.Copy(io.Discard, c)
+				// The server reads up to a marker, in our case `$`.
```

Comment:
can you copy-paste that comment before line #1716

Reply:
done